odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -239,6 +251,7 @@ PRINT TITLE="BANNO.TEMPLATE.V1.POW"
 
  IF @RGSTATE<>STATEEND THEN
   DO
+   NEWLINE
```

Comment:
Yep... shortly after this was when I went to the current methodology of checking for the BNOERRORCODE=0.  Do you think this should be upgraded to that logic?

Reply:
Probably not necessary but if you have a few free minutes, sure :).  I think it just is a little confusing since we don't generally use the state that way.  I definitely prefer the BNOERRORCODE check method.